playwright/plugins/docker/index.ts

```diff
@@ -22,22 +22,30 @@ import * as fse from "fs-extra";
 /**
  * @param cmd - command to execute
  * @param args - arguments to pass to executed command
+ * @param pipe - whether to pipe stdout & stderr to that of this process
```

Comment:
sortof feel like this might be better named "suppressOutput" with a `false` default, but whatever.

Reply:
Done